src/main/java/picard/illumina/CheckIlluminaDirectory.java

```diff
@@ -116,7 +116,7 @@
     protected int doWork() {
         final ReadStructure readStructure = new ReadStructure(READ_STRUCTURE);
         if (DATA_TYPES.isEmpty()) {
-            DATA_TYPES.addAll(Arrays.asList(IlluminaBasecallsConverter.DATA_TYPES_NO_BARCODE));
+            DATA_TYPES.addAll(Arrays.asList(BasecallsConverter.DATA_TYPES_WITH_BARCODE));
```

Comment:
Curious why this changed from `NO_BARCODE` to `WITH_BARCODE` and if that was intentional